devices/BlueNrg2/Aci/Gap.cs

```diff
@@ -0,0 +1,773 @@
+﻿using System;
+
+namespace BlueNrg2.Aci
+{
+	public class Gap
+	{
+		[Flags]
+		public enum EventMask : ushort
+		{
+			NoEvents = 0x0000,
+			LimitedDiscoverableEvent = 0x0001,
+			PairingCompleteEvent = 0x0002,
+			PassKeyReqEvent = 0x0004,
+			AuthorizationReqEvent = 0x0008,
+			SlaveSecurityInitiatedEvent = 0x0010,
+			BondLostEvent = 0x0020,
+			ProcCompleteEvent = 0x0080,
+			L2CapConnectionUpdateRequestEvent = 0x0100,
+			L2CapConnectionUpdateResponseEvent = 0x0200,
+			L2CapProcessTimeoutEvent = 0x0400,
+			AddressNotResolvedEvent = 0x0800
+		}
+
+		private readonly TransportLayer _transportLayer;
+
+		public Gap(TransportLayer transportLayer)
+		{
+			_transportLayer = transportLayer;
+		}
+
+		/// <summary>
+		///     Put the device in non-discoverable mode. This command disables the LL advertising.
```

Comment:
for the intellisense comments, only 1 space please so it does appear properly.
```suggestion
		/// Put the device in non-discoverable mode. This command disables the LL advertising.
```

Reply:
And please adjust elsewhere as well, it's in multiple places.
Tip, if you have multiple lines like this, you can make a vertical selection in Visual Studio and just delete all the columns extra spaces for a block at once ;-)